From 5f4c81bdf906c1214c65a2f7fab0d0473424bfca Mon Sep 17 00:00:00 2001
From: Michel Thebeau <michel.thebeau@windriver.com>
Date: Mon, 25 Jun 2012 13:23:10 -0400
Subject: [PATCH 2/5] ocf: OCF_CRYPTOSOFT depends on CRYPTO_HASH2

ERROR: "crypto_alloc_ahash" [crypto/ocf/cryptosoft.ko] undefined!
ERROR: "crypto_alloc_ablkcipher" [crypto/ocf/cryptosoft.ko] undefined!
ERROR: "crypto_ahash_digest" [crypto/ocf/cryptosoft.ko] undefined!
ERROR: "crypto_ahash_setkey" [crypto/ocf/cryptosoft.ko] undefined!

These are defined in crypto/ahash.c, which is enabled by CRYPTO_HASH2.

Signed-off-by: Michel Thebeau <michel.thebeau@windriver.com>
Signed-off-by: Bruce Ashfield <bruce.ashfield@windriver.com>
---
 crypto/ocf/Kconfig |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/crypto/ocf/Kconfig b/crypto/ocf/Kconfig
index 65a4461..83913fe 100644
--- a/crypto/ocf/Kconfig
+++ b/crypto/ocf/Kconfig
@@ -26,7 +26,7 @@ config OCF_CRYPTODEV
 
 config OCF_CRYPTOSOFT
 	tristate "cryptosoft (software crypto engine)"
-	depends on OCF_OCF
+	depends on OCF_OCF && CRYPTO_HASH2
 	help
 	  A software driver for the OCF framework that uses
 	  the kernel CryptoAPI.
-- 
1.7.10.4

